src/SourceBuild/Arcade/eng/common/templates/steps/source-build-build-tarball.yml

```
@@ -88,12 +88,12 @@ steps:
 
       # Use installer repo's NuGet.config during online testing to utilize internal feeds
       rm -f ${{ parameters.tarballDir }}/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/smoke-tests/online.NuGet.Config
-      cp NuGet.config ${{ parameters.tarballDir }}/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/smoke-tests/online.NuGet.Config
+      cp $(Build.SourcesDirectory)/NuGet.config ${{ parameters.tarballDir }}/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/smoke-tests/online.NuGet.Config
 
       dockerVolumeArgs="-v ${{ parameters.tarballDir }}:/tarball"
       dockerEnvArgs="-e EXCLUDE_OMNISHARP_TESTS=${{ parameters.excludeOmniSharpTests}}"
 
-      if [ '${{ parameters.isBootstrapped}}' != 'true' && '${{ parameters.installerBuildResourceId }}' != 'current' ]; then
+      if [ '${{ parameters.isBootstrapped }}' != 'true' && '${{ parameters.installerBuildResourceId }}' != 'current' ]; then
```

Would you mind correcting the syntax issue here while you are touching this step?  I think this should be using double `''`  This will fix this  issue we saw last week:

```
/mnt/vss/_work/_temp/1cec108b-8717-4dd5-8bda-81a1652d7995.sh: line 10: [: missing `]'
```

```suggestion
      if [[ '${{ parameters.isBootstrapped }}' != 'true' && '${{ parameters.installerBuildResourceId }}' != 'current' ]]; then
```